examples/fcma/file_io.py

```diff
@@ -178,3 +178,54 @@ def prepare_data(data_dir, extension, mask_file, epoch_file):
             (time2 - time1)
         )
     return raw_data, labels
+
+def generate_epochs_info(epoch_list):
+    """ use epoch_list to generate epoch_info defined below
+
+    Parameters
+    ----------
+    epoch\_list: list of 3D (binary) array in shape [condition, nEpochs, nTRs]
+        specification of epochs and conditions
+        assuming all subjects have the same number of epochs
+        len(epoch_list) equals the number of subjects
+        also assuming an epoch is always a continuous time course
+
+    Returns
+    -------
+    epoch\_info: list of tuple (label, sid, start, end)
+        the condition labels of the epochs
+        len(labels) labels equals the number of epochs
+        assuming the epochs of the same sid are adjacent
```

Comment:
Please write sentences in the docstrings. They are hard to follow without capitalization and punctuation.